lib/widgets/booking/bookings_stack.dart

```diff
@@ -60,10 +59,11 @@ class BookingsStack extends StatelessWidget {
           SizedBox(
             width: double.infinity,
             child: booking.isDisabled
-                ? DisabledBookingCard(
+                ? BookingCard(
                     key: Key(booking.id),
                     cabin: cabin,
                     booking: booking,
+                    isDisabled: true,
                   )
                 : BookingCard(
                     key: Key(booking.id),
```

Comment:
This `booking.isDisabled` check may also be simplified this way:

```dart
child: BookingCard(
  key: Key(booking.id),
  cabin: cabin,
  booking: booking,
  isDisabled: booking.isDisabled,
),
```

Reply:
Ooops, you're totally right. Silly mistake 🙈 
Updading it...